### DISCUSSION TITLE
I want to become a contributor

### DISCUSSION THREAD

**Discussion by jlshuman1961** [dotpcap/packetnet#161] [General]
I am completely new to GitHub and I want to make a contribution to the packetnet software to handle IGMPv3 packets.
I think I can figure out how to fork a branch that I can work on, but my biggest issue is getting the source to build with the appropriate tools (I am on Windows and will be using Visual Studio 2019) and .NET framework.
So my questions are:
1. Is there a Visual Studio solution file already in existence that I can download and use for building Packet.Net?
2.  What .NET framework is being used? (I looked at the PacketDotNet project file, and it appears to need 2 frameworks? netstandard2.0 AND net47?) How does that work? Which Frameworks do I need to install on my box? (version numbers).
Any help anyone can give me to get started will be much appreciated.

**Comment by kayoub5** [comment]
Install visual studio, Open the PacketNet.sln solution file.

if there is something missing visual studio will automatically prompt you to install it.
